internal/tools/exechelper/run.go

```diff
@@ -0,0 +1,115 @@
+// Package exechelper streamlines the running of external commands while both capturing and logging
+// their output.
+//
+// It builds on os/exec, expecting an instance of Cmd to manipulate.
+package exechelper
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base32"
+	"io"
+	"math/rand"
+	"os/exec"
+	"sync"
+	"time"
+
+	"github.com/pborman/uuid"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// OutputLogger allows custom logging of the run command output.
+type OutputLogger func(line string, logger log.FieldLogger)
+
+var encoding = base32.NewEncoding("ybndrfg8ejkmcpqxot1uwisza345h769")
```

Comment:
nit. Is that a constant maybe?

Reply:
Its just a random string that is used for encoding. We can put on a constant but I don't think it makes much sense, as it is not supposed to change.